.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -11,19 +11,30 @@ assignees: ''
 
 The following checklist covers all the steps for running release.
 
+First, a pre-release is published which is used to ensure all systems operate
+correctly in development environments. Once the pre-release is confirmed to be
+working, the final release is published.
+
 For more detail, refer to [`RELEASE.md`](https://github.com/infinyon/fluvio/blob/master/RELEASE.md)
 
 - [ ] Create a [new issue](https://github.com/infinyon/fluvio/issues/new?template=release_checklist.md) with this checklist template
 - [ ] Review all the [Getting Started](https://www.fluvio.io/docs/get-started/mac/) and [API](https://www.fluvio.io/api/) docs. Verify if they are up to date or will need updates
 - [ ] Inform team that release is about to begin
 - [ ] Ensure no merges are in flight
+
+## Pre-Release
+
+- [ ] Run the [`Release` workflow in Github Actions](https://github.com/infinyon/fluvio/actions/workflows/release.yml), make sure the `pre-release` check is set. (Retry at least once if failure)
+- [ ] Perform a pre-release test in a development environment
+
+## Stable Release
+
+If all systems are operational and no bugs are found in the pre-release, the final release can be published.
+
 - [ ] Create a PR for release
   - [ ] Update `VERSION` and `CHANGELOG.md` files (do not place a \n in the VERSION file, it breaks the CI)
   - [ ] Merge the PR
-- [ ] Run the [`Release` workflow in Github Actions](https://github.com/infinyon/fluvio/actions/workflows/release.yml) (Retry at least once if failure)  
-
-If you are doing a prelease stop here before publishing crates. Only run publish crates for the finalized release.
-
+- [ ] Run the [`Release` workflow in Github Actions](https://github.com/infinyon/fluvio/actions/workflows/release.yml), make sure the `pre-release` check is set. (Retry at least once if failure)
```

Comment:
`make sure the `pre-release` check is set` copy-paste typo?

Reply:
Good catch! Fixing!